paper-menu-button.html

```diff
@@ -93,6 +93,11 @@
         @apply(--paper-menu-button-content);
       }
 
+      .dropdown-content ::content > * {
+        /* Ensure items are full width so they don't wrap during animation. */
+        white-space: nowrap;
```

Comment:
I'd recommend against doing that.

The menu is only animated for a short time, not long enough to read the text of the long menu items. By the time someone can recognize that an ellipsis indicates that some of the menu item text is currently hidden, the complete menu item text will be shown. During that interval, showing ellipsis briefly would just constitute another source of flicker, when it's the intent of this PR to reduce flicker during the animation.

Moreover, ellipses are conventionally used at the ends of menu items with a specific semantic intent: to signal that a menu item will open a dialog. Applying ellipses automatically to arbitrary menu items could lead to confusion.